addons/website_event_meet/views/event_meeting_room_views.xml

```diff
@@ -87,4 +87,12 @@
             </search>
         </field>
     </record>
+
+    <record model="ir.actions.act_window" id="action_meeting_room_from_event">
```

Comment:
GuidelinesGuy: define id before model (record id="" model=""). Just because some ordering is necessary and we choose id a long time ago :)

Reply:
Done, corrected.